ecosystem/python/sdk/examples/multisig.py

```diff
@@ -0,0 +1,483 @@
+
+    if wait_for_user:
+        input("\nPress Enter to continue...")
+
+    # :!:>section_8
+    print("\n=== Signing rotation proof challenge ===")
+
+    rotation_proof_challenge = RotationProofChallenge(
+        sequence_number=0,
+        originator=deedee.address(),
+        current_auth_key=deedee.address(),
+        new_public_key=multisig_public_key.to_bytes(),
+    )
+
+    serializer = Serializer()
+    rotation_proof_challenge.serialize(serializer)
+    rotation_proof_challenge_bcs = serializer.output()
+
+    cap_rotate_key = deedee.sign(rotation_proof_challenge_bcs).data()
+
+    cap_update_table = MultiEd25519Signature(
+        multisig_public_key,
+        [
+            (bob.public_key(), bob.sign(rotation_proof_challenge_bcs)),
+            (chad.public_key(), chad.sign(rotation_proof_challenge_bcs)),
+        ],
+    ).to_bytes()
+
+    cap_rotate_key_hex = f"0x{cap_rotate_key.hex()}"
+    cap_update_table_hex = f"0x{cap_update_table.hex()}"
+
+    print(f"cap_rotate_key:   {cap_rotate_key_hex}")
+    print(f"cap_update_table: {cap_update_table_hex}")  # <:!:section_8
+
+    if wait_for_user:
+        input("\nPress Enter to continue...")
+
+    # :!:>section_9
+    print("\n=== Submitting authentication key rotation transaction ===")
+
+    from_scheme = Authenticator.ED25519
+    from_public_key_bytes = deedee.public_key().key.encode()
+    to_scheme = Authenticator.MULTI_ED25519
+    to_public_key_bytes = multisig_public_key.to_bytes()
+
+    entry_function = EntryFunction.natural(
+        module="0x1::account",
+        function="rotate_authentication_key",
+        ty_args=[],
+        args=[
+            TransactionArgument(from_scheme, Serializer.u8),
+            TransactionArgument(from_public_key_bytes, Serializer.to_bytes),
+            TransactionArgument(to_scheme, Serializer.u8),
+            TransactionArgument(to_public_key_bytes, Serializer.to_bytes),
+            TransactionArgument(cap_rotate_key, Serializer.to_bytes),
+            TransactionArgument(cap_update_table, Serializer.to_bytes),
+        ],
+    )
+
+    signed_transaction = rest_client.create_bcs_signed_transaction(
+        deedee, TransactionPayload(entry_function)
+    )
+
+    auth_key = rest_client.account(deedee.address())["authentication_key"]
+
+    print(f"Auth key pre-rotation: {auth_key}")
+
+    rest_client.submit_bcs_transaction(signed_transaction)
+
+    print("\nWaiting for API server to update...\n")
+    time.sleep(2.5)
+
+    auth_key = rest_client.account(deedee.address())["authentication_key"]
+
+    print(f"New auth key:         {auth_key}")
+    print(f"1st multisig address: {multisig_address}")  # <:!:section_9
+
+    if wait_for_user:
+        input("\nPress Enter to continue...")
+
+    # :!:>section_10
+    print("\n=== Publishing v1.0.0 ===")
+
+    packages_dir = "../../../../aptos-move/move-examples/upgrade_and_govern/"
+
+    command = (
+        f"aptos move compile --save-metadata "
```

Comment:
maybe move --save-metadata to anotehr line and make one per line

Reply:
@davidiw updated.